crates/javy/src/serde/de.rs

```diff
@@ -78,6 +96,21 @@ impl<'js> Deserializer<'js> {
         }
         unreachable!()
     }
+
+    /// When stringifying, circular dependencies are not allowed. This function
```

Comment:
I might be missing something. Does stringifying use the deserialization path in serde? In my head it's a serialization operation.

Reply:
It uses the deserializer, it's a bit counterintuitive, but we're deserializing from a JS value into any other format supported by serde.